### DISCUSSION TITLE
[Keynote] Hardware-Software Interface Specification for Verification in Accelerator-Rich Platforms

### DISCUSSION THREAD

**Discussion by rachitnigam** [cucapra/latte21#9] [Papers]
Author(s): Sharad Malik

**Comment by stephenneuendorffer** [comment]
Have you applied these techniques to less-embedded accelerators, such as HSA/ROCm accelerators?  Or distributed systems using RDMA for data movement?

**Reply by sm80171** [reply]
I think the interface level I am addressing is at a lower level than the above - if I understand this right, the ILA interface level is what the HSA driver would need to deal with, not the application that is layered above it.
